submissions/oluwatosinOke/react/src/App.js

```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
You are  not meant to use any css library for this project. Kindly remove and all instances of the its implementation in classes and js @Tocyn